sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -354,9 +355,14 @@ resource
     : identifier STRING
     ;
 
+insertStatement
+    : (ctes)? insertInto queryTerm queryOrganization                                       #singleInsertQuery
+    | (ctes)? fromClause multiInsertQueryBody+                                             #multiInsertQuery
+    ;
+
 queryNoWith
-    : insertInto? queryTerm queryOrganization                                              #singleInsertQuery
-    | fromClause multiInsertQueryBody+                                                     #multiInsertQuery
+    : queryTerm queryOrganization                                               #noWithQuery
+    | fromClause querySpecification queryOrganization                           #queryWithFrom
```

Comment:
Previously there can be multiple `multiInsertQueryBody`. Looks like there will be an Union for the inserts. Because the `insertInto` is optional, I think it can a `fromClause` and multiple `querySpecification queryOrganization`. But looks like the new rule doesn't allow it?

Reply:
@viirya Currently after this change, `fromClause` and multiple `querySpecification queryOrganization` will be handled by 2nd alternative of the `insertStatement` rule ?